bitcoin/src/blockdata/weight.rs

```diff
@@ -54,8 +54,13 @@ impl Weight {
         vb.checked_mul(Self::WITNESS_SCALE_FACTOR).map(Weight::from_wu)
     }
 
-    /// Constructs `Weight` from virtual bytes in const context.
-    pub const fn from_vb_const(vb: u64) -> Weight {
+    /// Constructs `Weight` from virtual bytes returning the unwrapped value.
```

Comment:
In 574efa798abed08145d7285a32305ba1316b1614:

I would drop "returning the unwrapped value". It is not obvious what is being unwrapped until you read the `Panics` section.

Reply:
Maybe just replace it with "panicking on onverflow".